ShopifySharp/Entities/SalesChannel/CreditCard.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ShopifySharp.Entities.SalesChannel
+{
+    public class CreditCard
```

Comment:
The `long`s in this class should be made nullable.

Reply:
Done